HealthcareSystem/PatientWebApp/wwwroot/html/add_feedback.html

```diff
@@ -72,7 +72,7 @@
 
         <!-- Send button -->
         <div class="row">
-            <input id="search_btn" type="submit" value="Send">
+            <input class="my_btn" type="submit" value="Send" style="margin-left: 45%;">
         </div>
```

Comment:
### Comment

I like how you make your code readable here! Well done. 

### Proposal

This HTML code description(comment)

```sh
 <!-- Send button -->
```

is a good practice, make more code like this!